packages/calling-component-bindings/src/providers/CallAgentProvider.tsx

```diff
@@ -53,4 +56,28 @@ export const CallAgentProvider = (props: CallAgentProviderProps): JSX.Element =>
  *
  * @public
  */
-export const useCallAgent = (): CallAgent | undefined => useContext(CallAgentContext)?.callAgent;
+export const useCallAgent = (): CallAgent | undefined => {
```

Comment:
Same comments about the thrown error and documentation as for `useCall` and `useTeamsCall`.